PKGBUILD

```diff
@@ -25,13 +24,16 @@ depends=('c-ares' 'intel-mkl' 'onednn' 'pybind11' 'openssl' 'lmdb' 'libpng' 'cur
 makedepends=('bazel' 'python-numpy' 'rocm-hip-sdk' 'miopen' 'rccl' 'git'
              'python-pip' 'python-wheel' 'python-setuptools' 'python-h5py'
              'python-keras-applications' 'python-keras-preprocessing'
-             'cython')
+             'cython' 'rocblas' 'rocrand' 'rocfft' 'hipfft' 'roctracer'
+             'hipsparse' 'hipsolver' 'rocsolver' 'rocm-hip-sdk')
 optdepends=('tensorboard: Tensorflow visualization toolkit')
-source=("$pkgname-$pkgver.tar.gz::https://github.com/tensorflow/tensorflow/archive/v${_pkgver}.tar.gz"
-        fix-c++17-compat.patch)
+source=("$pkgname::git+https://github.com/ROCmSoftwarePlatform/tensorflow-upstream.git"
```

Comment:
This is the release version of the package therefore this URL should not be changed.

Reply:
The we need to wait or i need to create a git version because the is no proper release yet. https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/tags

Reply:
Nice!, Should rocm-hip-sdk be removed as well since it is listed earlier also can you add roctracer and hipsolver next to the other rocm deps after rccl on the same line

Reply:
This PKGBUILD pulls from upstream tensorflow not from AMD's tensorflow fork. We should wait or add patches like u have done for hiprand.

Reply:
Even a git version of this PKGBUILD will use upstream tensorflow and not AMD's fork.
What you are suggesting is something like `tensorflow-rocm-amd-git`

Reply:
thx!